src/components/Footer.tsx

```diff
@@ -0,0 +1,29 @@
+"use client";
+import Link from "next/link";
+import Image from "next/image";
+import CsaLogo from "@/public/home/csalogo.svg";
+import { icons } from "@/data/footerData";
+
+const Footer = () => {
+  return (
+    <div className="relative bottom-0 flex h-24 w-full items-center justify-between border-t-8 border-csa-yellow-100 bg-csa-red-200 align-bottom">
```

Comment:
```suggestion
    <div className="flex items-center border-t-8 border-csa-yellow-100 bg-csa-red-200">
```
a lot of these classes don't do anything, also let's not define a hardcoded height value (h-24)